sequencer/src/main/java/net/consensys/linea/config/LineaProfitabilityCliOptions.java

```diff
@@ -57,6 +57,12 @@ public class LineaProfitabilityCliOptions implements LineaCliOptions {
       "--plugin-linea-extra-data-set-min-gas-price-enabled";
   public static final boolean DEFAULT_EXTRA_DATA_SET_MIN_GAS_PRICE_ENABLED = true;
 
+  public static final String PROFITABILITY_METRICS_BUCKETS =
+      "--plugin-linea-profitability-metrics-buckets";
+  public static final double[] DEFAULT_PROFITABILITY_METRICS_BUCKETS = {
+    0.9, 1.0, 1.2, 2, 5, 10, 100, 1000
```

Comment:
I think the min of 0.9 is too high for the defaults. I suggest 0.1, 0.3, 0.5, 0.7, 0.9, 1.0, 1.1, 1.2, 1.5, 2.0, 5.0, 10.0. @julien-marchand Other suggestions?

Reply:
Updated to the suggested list